xcp_d/workflows/postprocessing.py

```diff
@@ -290,42 +290,43 @@ def init_prepare_confounds_wf(
     ])
     # fmt:on
 
-    plot_design_matrix = pe.Node(
-        niu.Function(
-            input_names=["design_matrix", "temporal_mask"],
-            output_names=["design_matrix_figure"],
-            function=_plot_design_matrix,
-        ),
-        name="plot_design_matrix",
-    )
+    if params != "none":
```

Comment:
Amazing how many changes it takes to make xcpd not denoise. This looks good though